extensions/amp-addthis/validator-amp-addthis.protoascii

```diff
@@ -0,0 +1,73 @@
+#
+# Copyright 2018 The AMP HTML Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the license.
+#
+
+tags: {  # amp-addthis
```

Comment:
We try to keep these tagspecs in the same order and that order typically follows the order that is in the [proto](https://github.com/ampproject/amphtml/blob/master/validator/validator.proto#L385). You can see an [example here](https://github.com/ampproject/amphtml/blob/master/extensions/amp-timeago/validator-amp-timeago.protoascii).

The `html_format` shoudl come first, then `tag_name`, etc.